client/src/routes/Select.js

```diff
@@ -39,14 +38,16 @@ export const Floor = ({ mini, children: rooms }) => (
     ))}
```

Comment:
In the folded code block, we see here that the event handler creates the FSA and hands it off to `process`, thus allowing the `agent` to implement any consequences resulting from it.